wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/versions/Version.kt

```diff
@@ -0,0 +1,26 @@
+package it.krzeminski.githubactions.wrappergenerator.versions
+
+data class Version(val version: String) : Comparable<Version> {
+    val input: String = version.removePrefix("v").removePrefix("V")
+    val major = input.substringBefore(".").toIntOrNull() ?: 0
+    val minor = input.substringAfter(".").substringBefore(".").toIntOrNull() ?: 0
+    val patch = input.substringAfterLast(".").toIntOrNull() ?: 0
+    override fun compareTo(other: Version): Int {
```

Comment:
nit. some spacing before the function please:
```suggestion

    override fun compareTo(other: Version): Int {
```

Reply:
Done